var/spack/repos/builtin/packages/libtheora/package.py

```diff
@@ -88,7 +88,7 @@ def setup_build_environment(self, env):
         # devenv is needed to convert ancient MSbuild project to modern
         # msbuild project so MSBuild versions older than 2010 can build this
         # project
-        devenv_path = os.path.join(self.pkg.compiler.vs_root, "Common7", "IDE")
+        devenv_path = os.path.join(self.pkg["msvc"].vs_root, "Common7", "IDE")
```

Comment:
```suggestion
        devenv_path = os.path.join(self.pkg["c"].vs_root, "Common7", "IDE")
```
We already support the intel compilers on windows to some extent, but even if this isn't necessary now it's good future-proofing.

Reply:
It's not future proofing: not every compire will have a `vs_root` option (I guess "Visual Studio" root :slightly_smiling_face: )